Implemented log entries fetching command

Banana96 opened the pull request.

rogermb commented:
@TheHolyWaffle Yup, on it ^^

@Banana96 

Your PR looks good in general, but I have a few questions regarding what that command actually does.

* `lines` seems to be limited to the range 1 - 100, but this isn't mentioned anywhere.
* What happens if `lines` is omitted? Your JavaDoc mentions it would send the whole log file, but I have doubts about this, seeing how `lines` cannot exceed 100 if it's specified.
* `offset` (i.e. `begin_pos`) doesn't seem to do what the JavaDoc describes. It seems to only include the specified amount of bytes (or unicode code points? no clue) from the end of the file.
  * Example: Setting `begin_pos` to 1 sends back just one byte from the last log entry.
  * In other words, specifying `begin_pos` leads to tearing of the log entries
  * Is this really something we should expose to clients?
* `reverse` isn't mentioned anywhere, but it doesn't seem to be useful anyways - it only reverses the order of the entries sent back and doesn't actually start reading from the start of the log file, as one would expect.
* But what about `instance`? Shouldn't we also be able to read the server log file, not just the virtual server log?

Yay for TS3 ServerQuery API pitfalls, because the folks over at TeamSpeak have no idea how to design a good API.

Some minor points:

* You need to include a license header in `CLogView.java`. For legal reasons, we cannot accept unlicensed code into this code base.
** Just copy one of the headers from any other file and change the year and set your own name.
* Why are the parameters of `getLogEntries` ordered `int offset, int lines`, but CLogView uses `int lines, int offset`?
* Some other formatting stuff like missing newlines or whatever, but I'd happily fix those myself once this is pulled 😄

Banana96 commented:
@rogermb 
Issues that You pointed out should be fixed now. I dropped this "offset" thing, because it works a bit diffrent than I've expected, and I couldn't think about any of it's potential uses. Though, I've added a separation between fetching logs from master and virtual server file.

rogermb commented:
Merged! Thanks for your contribution :smile:

I'll probably still slightly adjust the JavaDoc a bit and I hope you can forgive me being a bit pedantic 😛 

Or at least that's what I intend to do once GitHub lets me push to `master` again...
